Fill league/flysystem bridge documentation (#143)
yokai-php/batch-src: docs/docs/bridges/league-flysystem.rst

Bridge with ``league/flysystem``
============================================================

> See `official documentation <https://flysystem.thephpleague.com>`__ on The League of Extraordinary Packages's website.

Reviewer: ```suggestion
Refer to the `official documentation <https://flysystem.thephpleague.com>`__ on The League of Extraordinary Packages's website.
```

Copy files job
------------------------------------------------------------

| This job will copy one or multiple files from a filesystem to another.

> | It need to be provided with a ``JobParameterAccessorInterface`` that will be asked to fetch what files to copy.

Reviewer: ```suggestion
| It must be provided with a ``JobParameterAccessorInterface`` which will be used to fetch the list of files to copy.
```

.. literalinclude:: league-flysystem/copy-files-job.php
   :language: php

.. seealso::
   | :doc:`What is an job? </core-concepts/job>`
   | :doc:`How do I access parameters of a job? </core-concepts/job-parameter-accessor>`

Move files job
------------------------------------------------------------

| This job will move one or multiple files from a filesystem to another.

> | It need to be provided with a ``JobParameterAccessorInterface`` that will be asked to fetch what files to move.

Reviewer: ```suggestion
| It must be provided with a ``JobParameterAccessorInterface`` which will be used to fetch the list of files to move.
```
